AUTHORS

```diff
@@ -96,4 +96,5 @@ Contributors (In alphabetical order):
   Vincent Untz <vuntz@gnome.org>
   Ville Skyttä <ville.skytta@iki.fi>
   Vojtech Trefny <vtrefny@redhat.com>
+  Vratislav Podzimek <vpodzime@redhat.com>
```

Comment:
Shall we put his actual e-mail address here? Or does this need to stay due to authorship since he did some work under a contract with RH? @vpodzime ?

Reply:
The list is generated from git log so we should use email addresses from the commits I guess.

Reply:
Right, sounds reasonable.